api/routes/apds/put.js

```diff
@@ -0,0 +1,48 @@
+const pick = require('lodash.pick');
+
+const logger = require('../../logger')('apds route put');
+const defaultApdModel = require('../../db').models.apd;
+const defaultUserModel = require('../../db').models.user;
+const loggedIn = require('../../auth/middleware').loggedIn;
+
+module.exports = (
+  app,
+  ApdModel = defaultApdModel,
+  UserModel = defaultUserModel
+) => {
+  logger.silly('setting up PUT /apds/:id route');
+  app.put('/apds/:id', loggedIn, async (req, res) => {
```

Comment:
From the [list of roles and activities](https://github.com/18F/cms-hitech-apd/blob/master/docs/users-and-roles.md) that @NAretakis identified, I'm wondering if this should be a `edit-document` activity.  This might be the first time we've had a real, high-level activity!